lib/quake_parser.ex

```diff
@@ -3,42 +3,120 @@ defmodule QuakeParser do
   `QuakeParser` is a parser for Quake game logs.
   """
 
-  alias QuakeParser.{Game, FindPlayers, KillsOfPlayer}
+  defstruct total_kills: 0, players: [], kills: %{}
 
+  @doc """
+  Parse the Quake log file, and return a QuakeParser object for each game.
+  """
+  @spec start(String.t()) :: list()
```

Comment:
```suggestion
  @spec start(String.t()) :: list(%__MODULE__{})
```

Ou então declara um type pro module. Tente sempre deixar mais explicito o conteúdo da list

Reply:
Ainda não entendi o motivo dessa função ser pública. Não dava pra ter uma única função no modúlo e.g. `report` que recebe um log e retorna o report?

Reply:
Sim, talvez eu n soube escolher o melhor nome, mas esse start devolve o log parseado, e pelo que entendi do [desafio](https://gist.github.com/akitaonrails/97310463c52467d2ecc6), eu tenho que ter uma função pra cada coisa, um pro log, um pras deaths e um pro scoreboard, que vou mandar em outro pr